src/libraries/System.Data.Common/src/System/Data/Common/DbException.cs

```diff
@@ -19,5 +19,6 @@ public abstract class DbException : System.Runtime.InteropServices.ExternalExcep
         protected DbException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) : base(info, context)
         {
         }
+        public virtual string? SqlState { get; }
```

Comment:
```suggestion
        public virtual string? SqlState => null;
```

To avoid generating `.field private initonly string '<SqlState>k__BackingField'` whose value would always be null.

Although perhaps matters little because you won't have many `DbException` instances in memory simultaneously.

Reply:
That's a good suggestion.